README.md

```diff
@@ -39,6 +39,34 @@ pnpm start
 
 Changes made to any source files in any package will make the demo rebuild and refresh. Since the projects are in the same repository and we use pnpm, we don't have to link them together.
 
+### Testing
+
+All new unit tests for components should be written using Jest and [React Testing Library](https://testing-library.com/), and not Enzyme which is being phased out.
+
+There are two packages that are tested:
+
+- `packages/react-vapor`: this is the source of all components so tests are critical here
+- `packages/e2eTesting`: Cypress E2E tests, which is relatively new
+
+To run all tests from the root, you can run `pnpm test`
+
+Alternatively, you can run it directly from the `react-vapor` package, which also allows using two other testing methods:
```

Comment:
Maybe `packages/react-vapor`? `react-vapor` makes me think about the root `react-vapor`.